### ISSUE TITLE
How to concatenate yolo model output to one tensor?

### ISSUE THREAD

**Issue by surajbharadwaj** [qqwweee/keras-yolo3#556]
Hello,

The yolo has three outputs. How to convert the model so that all these outputs are concatenated as one tensor or flattend to have [1 X N] array:

Eg:
[class:'bird',box1:'2',box2:'3',box3:'1.2',box4:'7.4',score:'89',class:'person',box1:'2.67',box2:'33',box3:'12.2',box4:'72.4',score:'91',class:'light',box1:'23',box2:'23',box3:'31.2',box4:'7.24',score:'93'....]
This is the expected output from tensorflow serving.
Want something like this.
is there any way to do that? flatten and reshape and concat???

**Comment by surajbharadwaj** [comment]
s = Reshape((1,), input_shape=classes_.shape)(tf.cast(classes_,tf.int32))
        c = Reshape((1,), input_shape=scores_.shape)(tf.cast(scores_,tf.int32))
        out = Concatenate(axis = 1)([tf.cast(boxes_,tf.int32),s,c])
        return out 

use this in decode yolo layer
